msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -162,4 +176,29 @@ void acquireToken(@NonNull final Activity activity,
         void onError(final MsalException exception);
     }
 
+    interface DeviceCodeFlowCallback{
+        /**
+         * Called to display verification uri, user code, and instruction message during device code flow.
+         *
+         * @param vUri verification uri
+         * @param userCode user code
+         * @param message instruction message
+         */
+        void getUserCode(@NonNull String vUri, @NonNull String userCode, @NonNull String message);
```

Comment:
Doesn't seem like it returns anything so I think a name that start with `get` is not suitable. How about `onUserCodeReceived`?

Reply:
This makes sense, I should've done this for `DeviceCodeFlowCommandCallback` in Common, too.

Reply:
@t-fadura You can open another PR in common to do the same (or do it as part of your next PR in common)

Reply:
I'd have to do a separate PR in common for it before this one is completed so I can change it in MSAL after it is changed in Common.